lib/registries/addon/components/make-decision-dropdown/template.hbs

```diff
@@ -48,6 +48,7 @@
             disabled={{not this.decisionTrigger}}
             local-class='SubmitButton'
             {{on 'click' (perform this.submitDecision)}}
+            {{on 'click' dd.close}}
```

Comment:
Was this tested?  This means clicking this button a second time closes the dropdown, which might cancel in-flight requests from the `submitDecision` task. Please use the `queue` helper from [ember-composable-helpers](https://github.com/CenterForOpenScience/ember-osf-web/blob/develop/lib/osf-components/addon/components/files/upload-zone/template.hbs#L15) 
```javascript
{{on 'click' (queue (perform this.submitDecision) dd.close)}}
```

Reply:
This is tested.